test_modules/test_module_2/test_module_2.info.yml

```diff
@@ -0,0 +1,6 @@
+name: 'NCM Search'
+description: 'Contains Search related configuration.'
+type: module
+core: 8.x
+dependencies:
+  - blocktabs:blocktabs
```

Comment:
**Add package information in custom module info file**
Generally project specific name as package should be added

 **References:**
 [Custom Module Development](https://www.drupal.org/docs/8/creating-custom-modules/let-drupal-8-know-about-your-module-with-an-infoyml-file)